apps/nextjs-website/src/_contents/webinars.ts

```diff
@@ -2,6 +2,28 @@ import { Webinar } from '@/lib/types/webinar';
 import { appIOGuideListsPath } from './appIo/guideListsPath';
 import { appIoQuickStartGuidePath } from './appIo/quickStartGuidePath';
 
+function imageFromPath(imagePath: string): {
+  readonly name: string;
+  readonly alternativeText: string | null;
+  readonly width: number;
+  readonly height: number;
+  readonly ext: string;
+  readonly mime: string;
+  readonly url: string;
+} {
```

Comment:
Following the DRY principle, what do you think about this alternative?

```suggestion
function imageFromPath(
  imagePath: string
): Required<Webinar['speakers'][0]['avatar']> {
```